pkg/fanal/analyzer/const.go

```diff
@@ -49,7 +49,8 @@ const (
 	TypeCargo      Type = "cargo"
 
 	// PHP
-	TypeComposer Type = "composer"
+	TypeComposer          Type = "composer"
+	TypeComposerInstalled Type = "composer-installed"
```

Comment:
How about `composer-vendor`?

```suggestion
	TypeComposerInstalled Type = "composer-vendor"
```

Reply:
`intalled.json` file placed in `path_to_app/vendor/...` dir.
So it make sense.
Updated.